storage/v2_coherent_read_session.py

```
@@ -0,0 +1,174 @@
+            read_v2_consensus_feature, validate_consensus_feature_args)
+        validate_consensus_feature_args(
+            symbol=symbol, market_type=market_type, timeframe=timeframe,
+            bucket_ts=bucket_ts, calculation_version=calculation_version)
+        return await read_v2_consensus_feature(
+            self._conn, symbol=symbol, market_type=market_type, timeframe=timeframe,
+            bucket_ts=bucket_ts, calculation_version=calculation_version)
+
+    async def fetch_v2_consensus_percentiles(
+        self, *, symbol: str, market_type: str, timeframe: str,
+        bucket_ts: datetime, calculation_version: str,
+    ) -> "tuple[Mapping, ...]":
```

Ruff (0.16.1)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)